utils/safeCommandsSet.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+function save_cd {
```

Comment:
```suggestion
function safe_cd {
```

I'm sorry, I introduce a typo here when suggesting the function name :see_no_evil:. It should be `safe_cd`.

Reply:
Ahh, got it, thanks!